packs/typeform/actions/get_results.py

```diff
@@ -0,0 +1,40 @@
+import urllib
+import httplib
+import requests
+
+from st2actions.runners.pythonrunner import Action
+
+__all__ = [
+    'TypeformAction'
+]
+
+
+class TypeformAction(Action):
+
+    def run(self, form_id, api_key, completed=True):
+        api_key = api_key if api_key else self.config['api_key']
+        completed = str(completed).lower()
+        url = "https://api.typeform.com/v0/form/%s?key=%s&completed=%s" % \
+            (form_id, api_key, completed)
+
+        headers = {}
+        headers['Content-Type'] = 'application/x-www-form-urlencoded'
+
+        params = {"key": api_key,
+                  "completed": completed
+                  }
+
+        data = urllib.urlencode(params)
+        response = requests.get(url=url,
+                                headers=headers, params=data)
+
+        if response.status_code == httplib.OK:
```

Comment:
If you want to avoid return True at the end, you should use return early programming style. So

```
if response.status_code != httplib.OK:
   blh blah blah
   raise Exception()
```

you don't need else here. this is the success path. 

Also, it might be helpful to do if response.status_code not in SUCCESS_CODES and define SUCCESS_CODES at class level. That way it's extensible.